keyboards/ploopyco/madromys/readme.md

```diff
@@ -0,0 +1,44 @@
+
+# Ploopyco Madromys Trackball
+
+It's a DIY, QMK-powered trackball!
+
+* Keyboard Maintainer: [PloopyCo](https://github.com/ploopyco)
+* Hardware Supported: RP2040
+* Hardware Availability: [Store](https://ploopy.co), [GitHub](https://github.com/ploopyco)
+
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile -kb ploopyco/madromys -km default
+
+# Triggering the Bootloader
+
+![bootloader pic](boot.jpg)
```

Comment:
```suggestion
```

Reply:
Committed in [41a834f](https://github.com/qmk/qmk_firmware/pull/21989/commits/41a834f9b1d993a40a52e9ae7e616de1dc879e84).